src/input/styles.scss

```diff
@@ -53,7 +53,6 @@
   border-block: styles.$control-border-width solid awsui.$color-border-input-default;
   border-inline: styles.$control-border-width solid awsui.$color-border-input-default;
   @include styles.font-body-m;
-  // IE requires input height to be explicitly set
```

Comment:
Removed this one, because `height` property below has been replaced by `block-size`, which is not supported by IE11 anyway